public/pages/DetectorConfig/containers/Features.tsx

```diff
@@ -0,0 +1,248 @@
+      showCodeModel: cloneShowCodeModal,
+    });
+  }
+
+  private getModalVisibilityChange = (index: number) => {
+    return this.state.showCodeModel[index];
+  };
+
+  private handleTableChange = (props: any) => {
+    this.setState({
+      sortField: props.sort.field,
+      sortDirection: props.sort.direction,
+    });
+  };
+
+  private sortedItems(items: Array<any>) {
+    let sorted = sortBy(items, this.state.sortField);
+    if (this.state.sortDirection == 'desc') {
+      sorted = sorted.reverse();
+    }
+    return sorted;
+  }
+
+  public render() {
+    const featureAttributes = get(this.props.detector, 'featureAttributes', []);
+
+    const sorting = {
+      sort: {
+        field: this.state.sortField,
+        direction: this.state.sortDirection,
+      },
+    };
+
+    const items = featureAttributes.map(
+      (feature: FeatureAttributes, index: number) => ({
+        name: feature.featureName,
+        definition: index,
+        state: feature.featureEnabled ? 'Enabled' : 'Disabled',
+      })
+    );
+
+    const sortedItems = this.sortedItems(items);
+
+    const columns = [
+      {
+        field: 'name',
+        name: 'Feature name',
+        sortable: true,
+      },
+      {
+        field: 'definition',
+        name: 'Feature definition',
+        render: (featureIndex: number) => {
+          const feature = featureAttributes[featureIndex];
+
+          const metaData = get(
+            this.props.detector,
+            `uiMetadata.features.${feature.featureName}`,
+            {}
+          );
+
+          if (
+            Object.keys(metaData).length === 0 ||
+            metaData.featureType == FEATURE_TYPE.CUSTOM
+          ) {
+            return (
+              <div>
+                <p>
+                  Custom expression:{' '}
+                  <EuiLink
+                    data-test-subj={`viewFeature-${featureIndex}`}
+                    onClick={() => this.showModal(featureIndex)}
+                  >
+                    View code
+                  </EuiLink>
+                </p>
+
+                {!this.getModalVisibilityChange(featureIndex) ? null : (
+                  <CodeModal
+                    code={JSON.stringify(feature.aggregationQuery, null, 4)}
+                    title={feature.featureName}
+                    subtitle="Custom expression"
+                    closeModal={() => this.closeModal(featureIndex)}
+                    getModalVisibilityChange={() =>
+                      this.getModalVisibilityChange(featureIndex)
+                    }
+                  />
+                )}
+              </div>
+            );
+          } else {
+            return (
+              <div>
+                <p>Field: {metaData.aggregationOf || ''}</p>
+                <p>Aggregation method: {metaData.aggregationBy || ''}</p>
+              </div>
+            );
+          }
+        },
+      },
+      {
+        field: 'state',
+        name: 'State',
+      },
```

Comment:
Maybe make this a sortable column as well?

Reply:
I considered that before.  Didn't do it because Lovinda's design does not show this is a sortable column.